mica/archive/aca_dark/__init__.py

```diff
@@ -1,2 +1,2 @@
 from dark_cal import *
-from dark_model import *
+from chandra_aca.dark_model import *
```

Comment:
In a perfect world, yes. But this would be a bit of a pain so I think we can just let it slide here.